trantor/net/EventLoopThread.h

```diff
@@ -59,10 +59,13 @@ class TRANTOR_EXPORT EventLoopThread : NonCopyable
     void run();
 
   private:
-    EventLoop *loop_;
+    // With C++20, use std::atomic<std::shared_ptr<EventLoop>>
+    std::shared_ptr<EventLoop> loop_;
+    std::mutex m_loop_;
```

Comment:
Use a name in consistence with drogon/trantor 's naming conventions. Such as `mutex_` or `loopMutex_`.

Reply:
Done